### PR TITLE
Move to Ruff

### PR DESCRIPTION
**Pull request by hettlage**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,11 +142,10 @@ However, `runPythonTests` does *not* generate the actual reports. For this you n
 
 This function can be used to run the following Python tools for testing:
 
-* bandit
-* black
-* flake8
-* isort
-* mypy
+* Bandit
+* Black
```
Comment (mofokeng-chaka) [inline README.md:146]: Isn't `black` part of `ruff`? The acknowledgements section of the documentation, states the following:

"Ruff's autoformatter is built on a fork of Rome's [rome_formatter](https://github.com/rome/tools/tree/main/crates/rome_formatter), and again draws on both the APIs and implementation details of [Rome](https://github.com/rome/tools), [Prettier](https://github.com/prettier/prettier), and [Black](https://github.com/psf/black)."

File: README.md [hunk 2]
```diff
@@ -161,7 +160,7 @@ RUN apt-get update -y
 RUN curl -sSL https://install.python-poetry.org | POETRY_HOME=/usr/local python3 -
 
 RUN pip install wheel
-RUN pip install allure-pytest bandit black flake8 isort mypy pytest
+RUN pip install allure-pytest bandit black mypy pytest ruff
```
Comment (mofokeng-chaka) [inline README.md:163]: See my comment above about `black`.

File: README.md (hunk 1 above)
Comment (hettlage) [reply README.md:146]: No, I don't think Black is part of Ruff. Quoting from the [FAQ](https://beta.ruff.rs/docs/faq/):

"As a project, Ruff is designed to be used alongside Black and, as such, will defer implementing stylistic lint rules that are obviated by autoformatting."

Black is also not mentioned as a tool that can be replaced with Ruff.

File: README.md (hunk 2 above)
Comment (hettlage) [reply README.md:163]: See my response. :-)

File: README.md (hunk 1 above)
Comment (mofokeng-chaka) [reply README.md:146]: Oh I see, thanks for making this clear.

**Pull request merged**